app/Console/Commands/AlertMACDStrategiesCommand.php

```diff
@@ -265,106 +272,119 @@ public function handle() {
+                                    $order["currency_price"] = $tmpAlert["currency_price"];
+                                    $order["user_id"] = $tmpAlert["user_id"];
+                                    $order["configuremacdbot_id"] = $tmpAlert["configuremacdbot_id"];
 
-                                if ($instrumentObj->alert_email != "") {
-                                    $orderObj = array('orderObject' => $order, 'instrumentPair');
+                                    if ($instrumentObj->alert_email != "") {
+                                        $orderObj = array('orderObject' => $order, 'instrumentPair');
 
-                                    $email = $instrumentObj->alert_email;
-                                    $alertType = $order["alert_type"];
+                                        $email = $instrumentObj->alert_email;
+                                        $alertType = $order["alert_type"];
 
-                                    $this->output->write("- EMAIL: " . $email, true);
+                                        $this->output->write("- EMAIL: " . $email, true);
 
-                                    $isSent = \Mail::send('emails.tradealertnew', ['data' => $orderObj], function ($m) use ($alertType, $email, $instrument) {
-                                                $m->from('no-reply@autobot.com', 'CryptoBee Trader');
-                                                $m->cc('parmaramit1111@gmail.com', 'Amit P');
-                                                $m->cc('scalableapplication@gmail.com', 'Arpit H');
+                                        $isSent = \Mail::send('emails.tradealertnew', ['data' => $orderObj], function ($m) use ($alertType, $email, $instrument) {
+                                                    $m->from('no-reply@autobot.com', 'CryptoBee Trader');
+                                                    $m->cc('parmaramit1111@gmail.com', 'Amit P');
+                                                    $m->cc('scalableapplication@gmail.com', 'Arpit H');
 
-                                                $subject = 'CryptoBee Trader: [' . $alertType . '] alert for ' . $instrument;
+                                                    $subject = 'CryptoBee Trader: [' . $alertType . '] alert for ' . $instrument;
 
-                                                $m->to($email)->subject($subject);
-                                            });
+                                                    $m->to($email)->subject($subject);
+                                                });
 
-                                    $this->output->write("- IS SENT?: " . $isSent, true);
+                                        $this->output->write("- IS SENT?: " . $isSent, true);
+                                    }
                                 }
                             }
                         }
                     }
+                    sleep(30);
```

Comment:
Why a sleep here?

Reply:
Sleep to avoid maximum request per minutes. There is limit for calling API every minutes